corehq/apps/campaign/models.py

```diff
@@ -33,16 +31,9 @@ class DashboardMapReport(models.Model):
         related_name='reports'
     )
     display_order = models.IntegerField(default=0)
-    report_configuration_id = models.CharField(max_length=126)
+    case_type = models.CharField(max_length=126)
```

Comment:
Looks like you went with the minimum of the different values we have for max_length for case types in different models :smiley_cat: 
Not sure how we came up with number 126, 255 seems more of a better default.

I assume we don't need `null=True` since that is the default anyway?

Reply:
> 255 seems more of a better default

Ooh, [CaseType.name has max_length=255]. While I think we should prevent anyone from setting a 255-character case type name :exploding_head: I agree we should make the fields match.